herddb-core/src/test/java/herddb/server/SimpleClientScanTest.java

```diff
@@ -247,36 +270,37 @@ private void checkCloseResultSetNotFullyScanned(
         }
     }
 
-    private void checkCloseResultSetNotFullyScannedWithTransactionAndAggregation(final HDBConnection connection, final Server server, boolean withTransaction) throws HDBException, ClientSideMetadataProviderException, InterruptedException {
+    private void checkCloseResultSetNotFullyScannedWithTransactionAndAggregation(final HDBConnection connection,
```

Comment:
this test case was already there, but it was not called, so this part was not tested.